app.js

```diff
@@ -0,0 +1,35 @@
+const Frame = require('./frame');
+const Score = require('./score');
+const prompt = require('prompt-sync')({sigint: true});
+
+class App {
+  constructor() {
+    this.roll_1 = 0;
+    this.roll_2 = 0;
+    this.score = new Score();
+  };
+
+  run() {
+    console.log('Welcome to bowling!');
+
+    for (let i = 1; i <= 10; i++) {
+      console.log(`\nFrame ${i}`)
+      this.roll_1 = prompt('Please enter the first number of knocked down pins: ');
+      this.roll_2 = prompt('Please enter the second number of knocked down pins: ');
+
+      this.addFrames(Number(this.roll_1), Number(this.roll_2));
+    }
```

Comment:
This could be broken out to another method so that your run() is simply descriptive